docs/en/05_Contributing/02_Release_Process.md

```diff
@@ -53,14 +53,14 @@ How to deprecate an API:
 *  Add a `@deprecated` item to the docblock tag, with a `{@link <class>}` item pointing to the new API to use.
 *  Update the deprecated code to throw a `[api:Deprecation::notice()]` error.
 *  Both the docblock and error message should contain the **target version** where the functionality is removed.
-   So if you're committing the change to a *3.1* minor release, the target version will be *4.0*. 
+   So, if you're committing the change to a *3.1* minor release, the target version will be *4.0*. 
 *  Deprecations should not be committed to patch releases
-*  Deprecations should just be committed to pre-release branches, ideally before they enter the "beta" phase.
+*  Deprecations should only be committed to pre-release branches, ideally before they enter the "beta" phase.
    If deprecations are introduced after this point, their target version needs to be increased by one.
 *  Make sure that the old deprecated function works by calling the new function - don't have duplicated code!
 *  The commit message should contain an `API` prefix (see ["commit message format"](code#commit-messages))
 *  Document the change in the [changelog](/changelogs) for the next release
-*  Deprecated APIs can be removed after developers had a chance to react to the changes. As a rule of thumb, leave the 
+*  Deprecated APIs can be removed after developers have had a chance to react to the changes. As a rule of thumb, leave the 
```

Comment:
Deprecated APIs can only be removed in MAJOR releases - do you mind updating this please?